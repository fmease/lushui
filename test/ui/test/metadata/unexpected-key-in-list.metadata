# TEST fail metadata

list: [first: 0, second: 2]
