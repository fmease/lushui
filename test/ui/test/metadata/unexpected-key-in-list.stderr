error: found ‘:’ but expected ‘,’
 --> ${TEST_FOLDER}/metadata/unexpected-key-in-list.metadata:1:13
  |
1 | list: [first: 0, second: 2]
  |             ^
  |

error: aborting due to previous error

